geopandas/tests/test_pandas_methods.py

```diff
@@ -582,3 +583,34 @@ def test_preserve_flags(df):
 
     with pytest.raises(ValueError):
         pd.concat([df, df])
+
+
+def test_concat(df):
+    df_rows = GeoDataFrame(pd.concat([pd.DataFrame(df), pd.DataFrame(df)], axis=0))
+    res1 = pd.concat([df, df], axis=0)
+    assert_geodataframe_equal(res1, df_rows)
+
+    # https://github.com/geopandas/geopandas/issues/1230
+    # Expect that concat should fail gracefully if duplicate column names belonging to
+    # geometry columns are introduced.
+    with pytest.raises(ValueError):
```

Comment:
Can you add a `match=` for the error message here? (to ensure it's actually the error from GeoPandas, and not another error)